pom.xml

```diff
@@ -2836,6 +2836,7 @@
                 <pattern>${maven.build.timestamp.format}</pattern>
                 <timeSource>current</timeSource>
                 <timeZone>America/Los_Angeles</timeZone>
+                <locale>en,US</locale>
```

Comment:
<img width="780" alt="image" src="https://github.com/apache/spark/assets/15246973/afc02a13-d494-4503-98df-a779d923fd45">
Because the default attribute of github is also this, setting this value is reasonable.
https://github.com/LuciferYang/spark/actions/runs/5274544416/jobs/9541917589#logs